doc/lua_api.txt

```diff
@@ -4375,6 +4375,12 @@ Call these functions only at load time!
 * `minetest.register_on_leaveplayer(function(ObjectRef, timed_out))`
     * Called when a player leaves the game
     * `timed_out`: True for timeout, false for other reasons.
+* `minetest.register_player_on_step(function(ObjectRef, dtime))`
+    * Called for every player on every server step.
+    * `ObjectRef`: The player.
+    * `dtime`: Time from last step (see also `minetest.register_globalstep`).
+    * Return `true` to avoid further `on_step`-callbacks to be called on this player (ie. if the
+      `ObjectRef` is no longer valid).
```

Comment:
This doesn't make sense, there is no situation in which this can happen.

Reply:
object ref is always valid in that case, we return only existing refs

Reply:
The mod can kick the player within this callback.

Reply:
yep but the ref is not invalidated on kick, it will be reclaimed later if i'm correct

Reply:
Oh, so, should I remove this and use callback mode `0`?

Reply:
Done.